src/access/examples/ExampleAccessControlDelegate.cpp

```diff
@@ -483,6 +512,99 @@ class EntryStorage
         index = found ? toIndex : ArraySize(acl);
     }
 
+public:
+    static constexpr uint8_t kTagInUse       = 1;
+    static constexpr uint8_t kTagFabricIndex = 2;
+    static constexpr uint8_t kTagAuthMode    = 3;
+    static constexpr uint8_t kTagPrivilege   = 4;
+    static constexpr uint8_t kTagSubjects    = 5;
+    static constexpr uint8_t kTagTargets     = 6;
+    static const size_t kStorageBufferSize   = 192;
```

Comment:
How was this value computed? Can we comment about why this is correct and sufficient (i.e. was this just a large value and serialize will validate the puts anyway?)

Reply:
I chose a large enough value. I've added a comment.